manager-parent/manager-core/src/main/resources/db/migration/0.2/V0.2.1__create_v2_db.sql

```diff
@@ -0,0 +1,75 @@
+CREATE TABLE BRIDGE_V2 (
+    id                 varchar(255) NOT NULL PRIMARY KEY,
+
+    customer_id        varchar(255) NOT NULL,
+    organisation_id    varchar(255) NOT NULL,
+    owner              varchar(255) NOT NULL,
+    subscription_id    varchar(255) NOT NULL,
+
+    name               varchar(255) NOT NULL,
+    cloud_provider     varchar(255) NOT NULL,
+    region             varchar(255) NOT NULL,
+
+    endpoint           varchar(255),
+    shard_id           varchar(255) NOT NULL,
+
+    submitted_at       timestamp NOT NULL,
+    published_at       timestamp,
+
+    error_id           integer,
+    error_uuid         varchar(255),
+
+    version            integer NOT NULL default 0,
+    generation         integer NOT NULL default 0,
+
+    unique (customer_id, name)
+);
+
+
+CREATE TABLE PROCESSOR_V2 (
+    id                 varchar(255) NOT NULL PRIMARY KEY,
+
+    name               varchar(255) NOT NULL,
+    flows              jsonb NOT NULL,
+
+    owner              varchar(255) NOT NULL,
+
+    submitted_at       timestamp NOT NULL,
+    published_at       timestamp,
+
+    version            integer NOT NULL default 0,
+    generation         integer NOT NULL default 0,
+
+    error_id           integer,
+    error_uuid         varchar(255),
+
+    bridge_id          varchar(255) NOT NULL,
+
+    unique (bridge_id, name),
+    constraint fk_bridge foreign key (bridge_id) references BRIDGE_V2 (id)
+);
+
+CREATE TABLE OPERATION (
+    id                   varchar(255) NOT NULL PRIMARY KEY,
+    type                 varchar(255) NOT NULL,
+    requested_at         timestamp NOT NULL,
+
+    managed_resource_id  varchar(255)
+);
+
+CREATE INDEX managed_resource_id_index ON OPERATION (managed_resource_id);
+
+CREATE TABLE CONDITION (
+    id                     varchar(255) NOT NULL PRIMARY KEY,
```

Comment:
Where do you get the field list from?

MGDOBR-1186 defines: `id`, `type`, `reason`, `message`, `lastUpdateTime`, `lastTransitionTime` and `status`.

You have `errorCode` and `component` too.

Reply:
Hey @manstis , the `Condition` in the CRD defines the `errorCode` so given that we said to have the same model I've reported it here (see https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/628d0eff3b3a28e64bec7351c8cefe4877660404/shard-operator-parent/shard-operator-core/src/main/java/com/redhat/service/smartevents/shard/operator/core/resources/Condition.java#L50). 

Regarding the `component`, I thought it was needed to store which is the `component`  (`controlPlane` or `dataPlane` - actually I'm going to refactor it with an ENUM in the java pojo, much better I think). wdyt?

Reply:
Thanks for the explanation @r00ta.

I forgot about `errorCode` (being `BridgeError` code.. IDK if renaming it for v2 would make it more obvious).

An enumeration sounds much better for `component` :+1: